trollmoves/client.py

```diff
@@ -179,21 +174,45 @@ def run(self):
                     beat_monitor(msg)
                     if msg.type == "beat":
                         continue
-
+                    # Handle public "push" messages as a hot spare client
+                    if msg.type == "push":
+                        # TODO: these need to be checked and acted if
+                        # the transfers are not finished on primary
+                        # client and are not cleared
+                        LOGGER.debug("Primary client published 'push'")
+                        add_to_ongoing(msg)
+
+                    # Handle public "ack" messages as a hot spare client
+                    if msg.type == "ack":
+                        LOGGER.debug("Primary client finished transfer")
+                        _ = add_to_file_cache(msg)
+                        _ = clean_ongoing_transfer(get_msg_uid(msg))
```

Comment:
why the `_`s ?

Reply:
Just to show that the functions return a value, but we don't care about them here. Could have also been a flake8 warning about that, but not sure.